lib/views/layout.erb

```diff
@@ -0,0 +1,29 @@
+<head>
+  <link rel="stylesheet" type="text/css" href="/style.css" media="screen" />
+  <link href='http://fonts.googleapis.com/css?family=Just+Me+Again+Down+Here' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+<div id="header">
+
+<h1>Chitter</h1>
+
+	<% if current_user %>
+		<div id="welcome-message">
+		Welcome, <%= current_user.email %>!
+		<form method="post" action="/sessions">
+			<input type="hidden" name="_method" value="delete">
+			<input type="submit" value="Sign Out" id="submit-buttons">
```

Comment:
When using a self-closing tag (i.e. you don't need to add an `</input>` to close it), add a trailing slash:

```html
<input ... ... ... /> <!-- <---- see how I closed it? -->
```